v3/src/components/common/edit-formula-modal.tsx

```diff
@@ -28,10 +29,14 @@ interface IProps {
 export const EditFormulaModal = observer(function EditFormulaModal({
   applyFormula, formulaPrompt, isOpen, onClose, titleInput, titleLabel, titlePlaceholder, value
 }: IProps) {
+  const minWidth = 400
+  const minHeight = 180
```

Comment:
These values appear here and in the CSS. Either import the values from the .css or use CSS variables, e.g. `--default-codap-modal-width` and `--default-codap-modal-height`, which are accessible from CSS and JS.